executor/common_linux.h

```diff
@@ -261,13 +268,20 @@ static void initialize_netdevices(void)
 				  "veth0_to_bridge", "veth1_to_bridge",
 				  "veth0_to_bond", "veth1_to_bond",
 				  "veth0_to_team", "veth1_to_team"};
+#endif
+#if SYZ_EXECUTOR || SYZ_ENABLE_NETDEV_DEVMASTERS
+	// If you extend this array, also update netdev_addr_id in vnet.txt.
```

Comment:
If DEVMASTERS is not enabled, then we won't have "veth0_to_bridge", "veth1_to_bridge", "veth0_to_bond", "veth1_to_bond", "veth0_to_team", "veth1_to_team" devices.